config/game_options.txt

```diff
@@ -93,7 +93,7 @@ PROBABILITY TRAITOR 5
 PROBABILITY TRAITORBRO 2
 PROBABILITY TRAITORCHAN 4
 PROBABILITY INTERNAL_AFFAIRS 3
-PROBABILITY NUCLEAR 2
+PROBABILITY NUCLEAR 3
```

Comment:
Changes Nukie probability to 3 from 2. This makes it the same chance as an IA round.